pkg/services/services_test.go

```diff
@@ -41,3 +46,544 @@ func TestGetHostnameCmd(t *testing.T) {
 		})
 	}
 }
+
+func TestHybridOverlayConfiguration(t *testing.T) {
```

Comment:
This PR is adding 550 lines of testing to test two potential string appends.
Please think carefully about this.
What value do these tests really add compared to the amount of  time it takes for people to review them, and the effort to change them when a new change needs to be made to the hybrid overlay configuration?